app/javascript/App/components/FilterPanel/filterComponents/Categories.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react'
 import Type from 'prop-types'
-import ItemSelector from '../../ItemSelectorAlt'
+import ItemSelectorCategory from '../../ItemSelectorCategory'
```

Comment:
I think "Category" is too specific for it to live in the root of the /components/ dir

Reply:
i'll reply to this one in comments of the PR